### DISCUSSION TITLE
Welcome to criticality_score Discussions!

### DISCUSSION THREAD

**Discussion by mudongliang** [hust-open-atom-club/OpenSift#5] [Announcements]
Discussions start here!

**Comment by zhuyanhuazhuyanhua** [comment]
1.目前我们分析元数据，我们未来会考虑代码语义级别的分析吗？
2.我们是否可以给优秀开源项目颁发徽章？
3.不同语言的漏洞挖掘和质量分析和函数调用链路分析不同，如果我们计划代码语义级别的分析，需要许多不同语言背景开源贡献者的鼎力相助

**Reply by mudongliang** [reply]
> 1.目前我们分析元数据，我们未来会考虑代码语义级别的分析吗？

我记得之前相智做一个项目的时候已经把程序调用关系考虑到里面了吧 @a2ure123 确认一下？

> 2.我们是否可以给优秀开源项目颁发徽章？ 

这个我可以有，但是目前感觉没啥用处

> 3.不同语言的漏洞挖掘和质量分析和函数调用链路分析不同，如果我们计划代码语义级别的分析，需要许多不同语言背景开源贡献者的鼎力相助

这个我同意，不过我建议可以尝试大模型的泛化性来帮忙
